test/spec/multivalue.wast

```diff
@@ -27,9 +27,9 @@
  )
 )
 
-(assert_return (invoke "pair") (tuple.make 2 (i32.const 42) (i64.const 7)))
-(assert_return (invoke "tuple-local") (tuple.make 2 (i32.const 0) (i64.const 0)))
-(assert_return (invoke "tuple-global-get") (tuple.make 2 (i32.const 0) (i64.const 0)))
-(assert_return (invoke "tuple-global-set"))
-(assert_return (invoke "tuple-global-get") (tuple.make 2 (i32.const 42) (i64.const 7)))
-(assert_return (invoke "tail-call") (tuple.make 2 (i32.const 42) (i64.const 7)))
+;; (assert_return (invoke "pair") (tuple.make 2 (i32.const 42) (i64.const 7)))
+;; (assert_return (invoke "tuple-local") (tuple.make 2 (i32.const 0) (i64.const 0)))
+;; (assert_return (invoke "tuple-global-get") (tuple.make 2 (i32.const 0) (i64.const 0)))
+;; (assert_return (invoke "tuple-global-set"))
+;; (assert_return (invoke "tuple-global-get") (tuple.make 2 (i32.const 42) (i64.const 7)))
+;; (assert_return (invoke "tail-call") (tuple.make 2 (i32.const 42) (i64.const 7)))
```

Comment:
Are these not valid? (If so why not remove them?)

Reply:
The wast parser does not parse `tuple.make`.

Reply:
Naively I'd expect the wast parser to parse a superset of the wat parser's input, which I guess is related to the other question about sharing logic between them?

Reply:
Since the goal is to run the upstream spec tests, it shouldn't be necessary to parse `tuple.make` to test multivalue. Rather than doing a bunch of work to support parsing `tuple.make` constants anyway, it seemed better to comment this out for now. I didn't delete them entirely because when we switch to the upstream spec tests, we will want to check that we regain this coverage.

Reply:
Fair enough. I'd have hoped to see `tuple.make` get parsed properly by reusing some wat parser logic, but as that's not the goal here it doesn't matter.